web/src/admin/stages/email/EmailStageForm.ts

```diff
@@ -202,18 +202,20 @@ export class EmailStageForm extends BaseStageForm<EmailStage> {
                         </p>
                     </ak-form-element-horizontal>
                     <ak-form-element-horizontal
-                        label=${msg("Token expiry")}
+                        label=${msg("Token expiration")}
                         ?required=${true}
                         name="tokenExpiry"
                     >
                         <input
-                            type="number"
-                            value="${first(this.instance?.tokenExpiry, 30)}"
+                            type="text"
+                            value="${first(this.instance?.tokenExpiry, "minutes=15")}"
```

Comment:
```suggestion
                            value="${first(this.instance?.tokenExpiry, "minutes=30")}"
```
To be consistent with the backend default